pytest.ini

```diff
@@ -1,9 +1,6 @@
 ##  This file holds default pytest configurations. This could also be removed and used directly with the command sent on CLI
 [pytest]
 addopts =
```

Comment:
Was there a reason for removal of these params?

Reply:
Same reason as [this](https://github.com/dimagi/dimagi-qa/pull/97#discussion_r879442918
)

Reply:
Yes. Got it!